hcx-apis/src/main/java/org/swasth/hcx/controllers/v1/ParticipantController.java

```diff
@@ -0,0 +1,98 @@
+package org.swasth.hcx.controllers.v1;
+
+import kong.unirest.HttpResponse;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.swasth.common.HttpUtils;
+import org.swasth.common.JsonUtils;
+import org.swasth.common.dto.ParticipantResponse;
+import org.swasth.common.dto.ResponseError;
+import org.swasth.hcx.controllers.BaseController;
+import org.springframework.http.HttpHeaders;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController()
+@RequestMapping(value = "/participant")
+public class ParticipantController  extends BaseController {
+    @Value("${registry.basePath}")
+    private String registryUrl;
+
+    @RequestMapping(value = "/create", method = RequestMethod.POST)
+    public ResponseEntity<Object> participantCreate(@RequestBody Map<String, Object> requestBody) throws Exception {
+        String url =  registryUrl + "/api/v1/Organisation/invite";
+        HttpResponse response = HttpUtils.post(url, JsonUtils.serialize(requestBody));
+        if (response != null && response.getStatus() == 200) {
+            Map<String, Object> result = JsonUtils.deserialize((String) response.getBody(), HashMap.class);
+            String participantCode = (String) ((Map<String, Object>) ((Map<String, Object>) result.get("result")).get("Organisation")).get("osid");
+            ParticipantResponse resp = new ParticipantResponse(participantCode);
+            return new ResponseEntity<>(resp, HttpStatus.OK);
+        } else if(response.getStatus() == 400) {
+            Map<String, Object> result = JsonUtils.deserialize((String) response.getBody(), HashMap.class);
```

Comment:
As in case of errors scenario, the code is same, please move code to a common method.

Reply:
Done